frontend/src/bundles/users/pages/profile.tsx

```diff
@@ -51,33 +62,39 @@ const Profile: React.FC = () => {
     );
 
     return (
-        <div className={styles.profile}>
-            <div className={styles.profile__container}>
-                <PageTitle title="Account settings" />
-                <ProfileCard title="Your plan">
-                    <ProfileInfo />
-                </ProfileCard>
-                <ProfileCard title="Account">
-                    <ProfileForm
-                        onSubmit={handleFormSubmit}
-                        user={{ email, firstName, lastName }}
-                    />
-                </ProfileCard>
-                <ProfileCard title="Social profile">
-                    <Socials
-                        socialMediaConnections={profile.socialMediaProfiles}
-                        dataStatus={profile.dataStatus}
-                        onSocialDisconnect={handleSocialMediaDisconnect}
-                    />
-                </ProfileCard>
-                <ProfileCard title="Email notifications">
-                    <Subscriptions />
-                </ProfileCard>
-                <ProfileCard title="Delete account">
-                    <DeleteAccount />
-                </ProfileCard>
+        <>
+            <Header>
+                <NavTabs items={headerItems}></NavTabs>
+                <UserProfile image={getUserAvatar(user)} />
+            </Header>
```

Comment:
If there were 100 pages, would we add header to each component?

Reply:
Done